### PR TITLE
New blog post draft

### PR DESCRIPTION
**Pull request by nuest**

### CODE REVIEW

**Comment by nuest** [comment]
Please squash the commits: https://github.com/blog/2141-squash-your-commits

File: _posts/2016-12-08-investigating-docker-and-R.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+[**googleComputeEngineR**](https://cloudyr.github.io/googleComputeEngineR/) provides an R interface to the Google Cloud Compute Engine API. It includes a function called docker_run that starts a Docker container in a Google Cloud VM and executes R code in it. Read [this article](https://cloudyr.github.io/googleComputeEngineR/articles/docker-ssh-futures.html) for details and examples. There are similar amibitions to implement docker capabilities in the [**analogsea package**](https://github.com/sckott/analogsea) that interfaces the Digital Ocean API.
+
+googleComputeEngineR abd analogsea use functions from the [**harbor package**](https://github.com/wch/harbor/) for R. You should have a look at it, because it may help you to control your own Docker containers that run either locally or in remote.
```
Comment (nuest) [inline _posts/2016-12-08-investigating-docker-and-R.md]: What does "abd analogsea" mean?

**Comment by nuest** [comment]
https://github.com/MatthiasHinz/o2r-project.github.io/pull/1

**Comment by nuest** [comment]
Two tasks left:

- [ ] please check the HTML check output and fix the links: https://travis-ci.org/o2r-project/o2r-project.github.io/builds/184246060: `Link: http://sites.duke.edu/researchcomputing/files/2014/09/mccahill-DockerDays.pdf Redirected to: https://sites.duke.edu/researchcomputing/files/2014/09/mccahill-DockerDays.pdf`
- [ ] update the filename to today with `git mv`

**Comment by MatthiasHinz** [comment]
Can you check the boxes for me? ;)

**Comment by nuest** [comment]
Please try again, I invited you to the project.

**Comment by MatthiasHinz** [comment]
Works now! Thanks.

**Pull request merged**